src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -433,7 +433,13 @@ namespace winrt::TerminalApp::implementation
                         THROW_IF_FAILED(dialog->SetDefaultExtension(L"txt"));
 
                         // Default to using the tab title as the file name
-                        THROW_IF_FAILED(dialog->SetFileName((control.Title() + L".txt").c_str()));
+                        auto filename = { control.Title() };
+                        char unallowed_chars[] = "<>:\\/\"|?*";
+                        for (unsigned int i = 0; i < strlen(unallowed_chars); ++i)
+                        {
+                            filename.erase(std::remove(filename.begin(), filename.end(), unallowed_chars[i]), filename.end());

```

Comment:
We're on C++20 which allows you to replace the erase-remove idiom with the more modern `std::erase`:
```cpp
std::erase(filename, unallowed_chars[i]);
```

Reply:
Maybe could be useful to put this information in the contribution guide, what do you think? So other starting developers like me know what language standard is in use and use the correct features: The development section could be a good place

Reply:
You can always check C++ version in project properties.

Reply:
Thank you! Didn't know about it